package.json

```diff
@@ -277,6 +277,24 @@
                 "command": "jupyter.runAndDebugCell",
                 "key": "ctrl+alt+shift+enter",
                 "mac": "ctrl+shift+enter"
+            },
+            {
+                "command": "jupyter.runByLine",
+                "key": "f10",
+                "mac": "f10",
```

Comment:
Out of curiosity is the 'mac' entry necessary here? I assumed 'key' was sufficient.

Reply:
IDK, I don't have a mac to test it.